Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -316,7 +316,12 @@ public virtual void RenderCustomInspector()
                             EditorGUI.indentLevel = indentOnSectionStart + 3;
 
                             string prefKey = themeItem.objectReferenceValue.name + "Profiles" + i + "_Theme" + t + "_Edit";
+                            bool hasPref = EditorPrefs.HasKey(prefKey);
                             bool showSettings = EditorPrefs.GetBool(prefKey);
+                            if (!hasPref)
```

Comment:
This also seems like an unrelated change to the PR

Reply:
This was a bug fix made as I was working on the themes, every time a new theme is added the properties were closed by default, it was getting annoying.

Reply:
I'll add to the bug fix section.